### ISSUE TITLE
PeoplePicker: No way to tell user how to choose an item in suggestion list.

### ISSUE THREAD

**Issue by MirandaGe** [OfficeDev/office-ui-fabric-react#1012]
### Bug Report

- __Package version(s)__: any
- __Browser and OS versions__: IE
- test with JAWS: 18.0.2324

#### Describe the issue:

1. input some letter in the input area of people picker
2. the auto-suggestion list shown, but not mention how to choose an item in the list. (using enter or space?)

#### Expected behavior:

mention how to add the item to picker.

### If applicable, please provide a codepen repro:

I tested on fabric-react demo page: https://dev.office.com/fabric#/components/peoplepicker

**Comment by battletoilet** [comment]
@MirandaGe are you seeing this issue while using a screen reader?

**Comment by MirandaGe** [comment]
Yes, I'm using JAWS 18. I updated the information above now.

**Comment by joschect** [comment]
@MirandaGe There needs to be support for all aria tags to get passed through to the input element. I'll look into getting that added.

**Comment by YusongLiu** [comment]
Hi @joschect, is there some update?

**Comment by joschect** [comment]
@YusongLiu Hi, I'm sorry but I've gotten pulled into some other things and haven't had a chance to look into this more.

**Comment by machineloop** [comment]
We're seeing some similar accessibility issues with PeoplePicker:

- it doesn't announce when a user is selected/added
- doesn't announce when a user is removed/deleted
- doesn't announce when the list gets updated
- doesn't announce when there's no results found.

This is with EDGE+Narrator; seeing similar w/ JAWS+IE11. Should I create a new issue, or are these closely enough related to fall under this issue?

**Comment by dzearing** [comment]
This was apparently fixed with the merges above. Closing.
